definitions/procedures/backup/config_files.rb

```diff
@@ -20,7 +20,7 @@ def run
         configs = config_files.join(' ')
         execute!("tar --selinux --create --gzip --file=#{tarball} " \
           "--listed-incremental=#{increments} --ignore-failed-read " \
-          "#{configs}")
+          "#{configs}", :valid_exit_statuses => [0, 1])
```

Comment:
@pmoravec Is it safe? I don't care much about online backup. But it could let some inconsistency sneak in offline backup. I guess that doing config backup after we stop services (https://github.com/theforeman/foreman_maintain/pull/248) might help but still there is a chance of something unexpected. 
Could we make this change optional by a new parameter that is set for online backups only?
Does it make sense or I'm I too paranoid?

Reply:
It makes sense to be tolerant to changed configs _only_ on online backups - will update the PR.

(that's why I am not a great developer, I fix just my (broken) call flow)